provider/provider_dotnet_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2016-2024, Pulumi Corporation.  All rights reserved.
+
+//go:build !go && !yaml && !nodejs && !python && !java
+// +build !go,!yaml,!nodejs,!python,!java
+
+package provider
+
+import (
+	"github.com/stretchr/testify/require"
+	"path/filepath"
+	"testing"
+)
+
+func TestGameLift(t *testing.T) {
```

Comment:
I know the example given was in dotnet, but do we want to introduce another place to create tests? Since it is not an issue specific to dotnet I think we should just add it to our existing tests.

Unless you think it is useful to have dotnet tests, i'll leave it up to you.